src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -292,16 +382,16 @@ class ExternalDatabaseManagementService(
         deleteOrganizationExternalDatabaseTables(orgId, tableIdByFqn)
 
         //drop db from schema
-        val dbName = acm.getOrganizationDatabaseName(orgId)
-        acm.connect(dbName).connection.use { conn ->
+        val dbName = externalDbManager.getOrganizationDatabaseName(orgId)
+        externalDbManager.connect(dbName).connection.use { conn ->
```

Comment:
might be worth killing all connections to the database before executing DROP DATABASE

Reply:
🧐

Reply:
currently it's `dropAllConnectionsToDatabaseSql` in `AssemblerConnectionManager` -- this is called right now before an org database rename since the rename will fail if there are any connections to the database (including cached ones from our own services + indexer, I think) but also if there are any users connected to the database. Seems possible for the same error to arise here -- you could just execute an extra update using the statement, like

```
externalDbManager.connect(dbName).connection.use { conn ->
  conn.createStatement().use { stmt ->
    stmt.execute(dropAllConnectionsToDatabaseSql(dbName))
    stmt.execute("DROP DATABASE $dbName")
  }
}
```

Reply:
i see you did not change this code, but deleting organizations has been broken and this may be related? ¯\_(ツ)_/¯

Reply:
ah yes, this is sensible